app/models/problem_check.rb

```diff
@@ -28,14 +28,45 @@ class ProblemCheck
   #
   config_accessor :max_blips, default: 0, instance_writer: false
 
+  # Problem check classes need to be registered here in order to be enabled.
+  #
+  # Note: This list must come after the `config_accessor` declarations.
+  #
+  CORE_PROBLEM_CHECKS = [
+    EmailPollingErroredRecently,
```

Comment:
Just one question; should we prefix all of these with `ProblemCheck::` to be explicit? Might not be something to worry about, was just thinking about possible clashes.